src/polytope/models/ProblemCheckerTest.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+
+
+class ProblemCheckerVerdict(Enum):
+    """! Verdicts of problem checkers enumeration class."""
+
+    # Output is correct.
+    Correct = auto()
```

Comment:
(enum) Don't we need to cover up all outputs by testlib.h? (partial answer, etc...)

Reply:
Since the primary goal of the project is to support the setting of common batch-style problems, I intentionally ignored them.
Would it be better to think ahead and cover them up?

Reply:
I agree with you. Also, as i remember the partial score behavior of testlib is target of migration -- so it seems reasonable.
But at least, considering the case user copy-paste the alien checker from elsewhere, how about adding `UnsupportedResult` type?

Reply:
Thanks for the advice. I did add it.

https://github.com/Studio-Polytope/Polytope/blob/47ec41d2bdfa27c385baa7baa85319c230809df4/src/polytope/models/ProblemCheckerTest.py#L5-L18